client/components/theme-tier/constants.js

```diff
@@ -5,7 +5,22 @@ import {
 	PLAN_PREMIUM,
 	getPlan,
 } from '@automattic/calypso-products';
-import { translate } from 'i18n-calypso';
+import { englishLocales } from '@automattic/i18n-utils';
+import i18n, { translate } from 'i18n-calypso';
+
+const getIncludedWithLabel = ( planSlug ) => {
+	const localeSlug = i18n.getLocaleSlug();
+
+	const shouldShowNewString =
+		( localeSlug && englishLocales.includes( i18n.getLocaleSlug() ) ) ||
+		i18n.hasTranslation( 'Included with %(planName)s' );
+
+	return shouldShowNewString
+		? translate( 'Included with %(planName)s', {
+				args: { planName: getPlan( planSlug )?.getTitle() },
```

Comment:
What happens if `getPlan` can't invoke `getTitle`?

Reply:
Same as before, the plan name would be empty, so the option would read as "Include with".

Reply:
Anyway, don't worry about making this helper function too robust, since it's intended to be deleted as soon as the new translations are ready.